signer/gpg2/gpg2_test.go

```diff
@@ -231,7 +315,153 @@ func TestSignData(t *testing.T) {
 				}
 				t.Logf("GnuPG PGP signature verification output:\n%s\n", out)
 			})
+		})
+	}
+}
+
+func TestGPG2Signer_SignFiles(t *testing.T) {
```

Comment:
Is there verification of `os.RemoveAll()` running successfully within `SignFiles`?